### PR TITLE
Update Documentation For Upgrading Django

### PR DESCRIPTION
**Pull request by dchukhin**
This pull request updates our documentation for upgrading Django, and includes both the updated reStructuredText and HTML files.

### CODE REVIEW

File: source/upgrading-django.rst [hunk 1]
```diff
@@ -5,39 +5,40 @@ Django routinely releases new versions containing bug and
...
-major version at a time, working through all the upgrade notes for
-one release and resolving all issues before moving on to the
-next Django major version.
 
-It's okay to jump straight to the last available minor release
-of a major version though.
+Helpful Principles
+------------------
+
+Upgrade from LTS version to LTS version
```
Comment (vkurup) [inline source/upgrading-django.rst]: I like this change. One question... the text here assumes that you are currently on an LTS version. Should we add a sentence along the lines of "If you're not on an LTS, upgrade to the nearest LTS first"?

File: source/upgrading-django.rst [hunk 2]
```diff
@@ -80,3 +81,88 @@ of the package I ended up with, and update the requirements file
...
+
+   and see the same list in a column format with::
+
+       pip list --outdated --format=columns
+
+4. Once all of the deprecation warnings have been resolved,
+   update the requirements file with the new version of Django
+   and run ``pip install -r requirements.txt``.
+
+5. Try to run the server locally with ``python manage.py runserver``,
```
Comment (vkurup) [inline source/upgrading-django.rst]: Most of our sites need to run `npm somethin somethin` to work, but I think the dev should understand that. (or you could remove the `manage.py` command and leave it as an exercise for the reader to figure out how to run the server locally.)

File: source/upgrading-django.rst (hunk 1 above)
Comment (dchukhin) [reply source/upgrading-django.rst]: Sounds like a good idea

File: source/upgrading-django.rst (hunk 2 above)
Comment (dchukhin) [reply source/upgrading-django.rst]: Heh, I remember the '... will be left as an exercise for the reader' text from textbooks always being unsatisfactory.

**Pull request merged**